Sources/Core/Extensions/Foundation/Dictionary.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Dictionary+Povio.swift
+//  PovioKit
+//
+//  Created by Borut Tomazin on 21/09/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import Foundation
+
+public extension Dictionary {
+  /// Make union from `Self` and given `dictionary`
+  /// Examples:
+  /// ```
+  /// print([1: "One", 2: "Two"].union([3: "Three", 4: "Four"]) // result: [1: "One", 2: "Two", 3: "Three", 4: "Four"]
+  /// print([1: "One", 2: "Two"].union([2: "Two"]) // result: [1: "One", 2: "Two"]
+  /// ```
+  func union(_ dictionary: Dictionary) -> Dictionary {
```

Comment:
I believe Dictionary already has something similar implemented:

- https://developer.apple.com/documentation/swift/dictionary/merge(_:uniquingkeyswith:)-7smbb
- https://developer.apple.com/documentation/swift/dictionary/merging(_:uniquingkeyswith:)-3vtfs

Reply:
It's been there since iOS 8 🤦‍♂️ 
Thanks, Toni. Way better yes since it handles merges with different values.